ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceNotifier.java

```diff
@@ -56,7 +55,7 @@
 
   private Optional<ForkChoiceState> lastSentForkChoiceState = Optional.empty();
   private Optional<PayloadAttributes> lastSentPayloadAttributes = Optional.empty();
-  private Optional<Bytes8> lastPayloadId = Optional.empty();
+  private Optional<SafeFuture<Optional<Bytes8>>> lastFuturePayloadId = Optional.empty();
```

Comment:
I think I'd make this just `SafeFuture<Optional<Bytes8>> lastFuturePayloadId = SafeFuture.completedFuture(Optional.empty());